### PR TITLE
Feat/image search

### PR DESCRIPTION
**Pull request by SavvyShah**

### CODE REVIEW

File: stories/index.js [hunk 1]
```diff
@@ -2481,6 +2562,86 @@ storiesOf("Search components/SearchBox", module)
...
+      }}
+      themePreset={select("themePreset", ["light", "dark"])}
+    />
+  ))
+  .add(
+    "With document suggestions",
+    () => (
+      <SearchBoxRSDefault
+        placeholder="Search Books..."
+        enableIndexSuggestions={boolean('enableIndexSuggestions', false)}
```
Comment (mohdashraf010897) [inline stories/index.js:2585]: @SavvyShah Let's remove the `enableIndexSuggestions` knob as it seems not very relevant to the story

File: stories/index.js [hunk 2]
```diff
@@ -2481,6 +2562,86 @@ storiesOf("Search components/SearchBox", module)
...
+          size: number("size", 5),
+          sectionLabel: text("sectionLabel","🕝 Recent Document Suggestions")
+        }}
+        includeFields={[
+            "original_title",
+            "original_language",
+            "overview"
+        ]}
+        showClear
+        renderItem={(suggestion)=>(<div>{suggestion._source.original_title} <span style={{color: "crimson", backgroundColor: "lightgray", padding: 5, borderRadius: 3}}>{new Date(Number(suggestion._source._timestamp) * 1000).toDateString() || ""}</span></div>)}
```
Comment (mohdashraf010897) [inline stories/index.js]: @SavvyShah Let's have this hidden inside the SearchBoxDefault wrapper component to improve the readability. You can use a custom prop to switch the `renderITem` for documentsuggestions on/ off, a suggestion is `shouldCustomRenderDocumentSuggestion`

File: stories/index.js [hunk 3]
```diff
@@ -2481,6 +2562,86 @@ storiesOf("Search components/SearchBox", module)
...
+          size: number("size", 5),
+          sectionLabel: text("sectionLabel","🕝 Recent Document Suggestions")
+        }}
+        includeFields={[
+            "original_title",
+            "original_language",
+            "overview"
+        ]}
+        showClear
+        renderItem={(suggestion)=>(<a style={{width:'100%', height: '100%', display: 'block', textDecoration: "none", color: "inherit"}} href={`https://www.google.com/search?q=${suggestion._source.original_title}`} target="_blank" rel="noreferrer">{suggestion._source.original_title}</a>)}
```
Comment (mohdashraf010897) [inline stories/index.js]: @SavvyShah refer to the just previous comment

**Review by mohdashraf010897 — CHANGES_REQUESTED** [review]

File: stories/index.js (hunk 2 above)
Comment (SavvyShah) [reply stories/index.js]: I've moved them in a utils folder.

File: stories/index.js (hunk 3 above)
Comment (SavvyShah) [reply stories/index.js]: I've moved them in a utils folder.

File: stories/index.js (hunk 1 above)
Comment (SavvyShah) [reply stories/index.js:2585]: It's somewhat relevant for document suggestions, if we want to hide index suggestions when we already have some document suggestions. The change is not a new change although it appears in green.

**Pull request merged**